docs/overview.rst

```diff
@@ -116,31 +165,34 @@ permission settings.
 What follows is a comparison table showing how Kinto stacks up compared to some
 other projects in this space.
 
-===========================  ======  ======  ========  =======  ==============
-Project                      Kinto   Parse   Firebase  CouchDB  Remote-Storage
----------------------------  ------  ------  --------  -------  --------------
+===========================  ======  ======  ========  =======  ==============  =======
+Project                      Kinto   Parse   Firebase  CouchDB  Remote-Storage  Kuzzle
+---------------------------  ------  ------  --------  -------  --------------  -------
+Offline-first client         ✔       ✔       ✔         ✔        ✔
 Fine-grained permissions     ✔       ✔       ✔
-Easy query mechanism         ✔       ✔       ✔         [#]_     [#]_
-Conflict resolution          ✔       ✔       ✔         ✔        ✔ [#]_
-Validation                   ✔       ✔       ✔         ✔
-Revision history                                       ✔
-File storage                         ✔                 ✔        ✔
-Batch/bulk operations        ✔       ✔                 ✔
-Changes stream               [#]_    ✔       ✔         ✔
-Pluggable authentication     ✔                         ✔        [#]_
-Pluggable storage / cache    ✔                                  ✔
-Self-hostable                ✔                         ✔        ✔
-Decentralised discovery      [#]_                               ✔
-Open source                  ✔                         ✔        ✔
-Language                     Python                    Erlang   Node.js [#]_
-===========================  ======  ======  ========  =======  ==============
+Easy query mechanism         ✔       ✔       ✔         [#]_     [#]_            ✔
+Conflict resolution          ✔       ✔       ✔         ✔        ✔ [#]_          ?
+Validation                   ✔       ✔       ✔         ✔                        ?
+Revision history                                         ✔
+File storage                 ✔ [#]_   ✔                  ✔        ✔
+Batch/bulk operations        ✔       ✔                  ✔
+Changes stream               ✔ [#]_   ✔       ✔         ✔                        ✔
+Pluggable authentication     ✔                          ✔        [#]_
+Pluggable storage / cache    ✔                                    ✔              ✔
+Self-hostable                ✔                          ✔        ✔              ✔
+Decentralised discovery      [#]_                                 ✔
+Open source                  ✔                          ✔        ✔              ✔
+Language                     Python                    Erlang   Node.js [#]_    Node.js
+===========================  ======  ======  ========  =======  ==============  =======
```

Comment:
I'd probably list live notifications support for all solutions, mentioning the feature is upcoming through an upcoming plugin for kinto.